diagnostic.md

```diff
@@ -1,10 +1,20 @@
 ## Question 1
 
+Comfort: 2
+Clarity: 2
+I used google, and google, and google, because google
+Not super comfy with using jquery in dev tools, but I think i understand using it in
+html.
+
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
 <!-- your answer starts here -->
-
+<div id="js-pjax-loader-bar" class="pjax-loader-bar"><div class="progress"></div></div>
+<div id="start-of-content" class="accessibility-aid"></div>
+<div class="facebox" id="facebox" style="display:none;">
+<div class="container site-footer-container">
+Why can't I do $$('css-selectors')? Am I missing a step?
```

Comment:
You're on the right track. The actual select would be `.pjax-loader-bar` for example, or `.facebox`. Please review this topic: http://www.w3schools.com/jquery/jquery_ref_selectors.asp